src/Microsoft.ML.Transforms/NAIndicatorTransform.cs

```diff
@@ -68,116 +78,196 @@ private static VersionInfo GetVersionInfo()
         internal const string FriendlyName = "NA Indicator Transform";
         internal const string ShortName = "NAInd";
 
-        private static string TestType(ColumnType type)
+        internal static string TestType(ColumnType type)
         {
             // Item type must have an NA value. We'll get the predicate again later when we're ready to use it.
             Delegate del;
             if (Conversions.Instance.TryGetIsNAPredicate(type.ItemType, out del))
                 return null;
             return string.Format("Type '{0}' is not supported by {1} since it doesn't have an NA value",
-                type, LoaderSignature);
+                type, LoadName);
+        }
+
+        public class ColumnInfo
```

Comment:
>c class [](start = 13, length = 7)

`sealed`. Also, why do you need it? The base class is doing fine with just a `(string input, string output)` value tuple